docs/configfile.adoc

```diff
@@ -0,0 +1,47 @@
+= Yaml Configuration file
+
+Options can be configured by a YAML file using `-config`:
```

Comment:
I think it would be helpful to document that the configuration file uses [lower camel case](https://en.wikipedia.org/wiki/Camel_case) naming. For example, `-jar-cache` is `jarCache:` in the configuration file. Most of the command-line arguments are already in camel case, but `-jar-cache` is an exception.

Reply:
Ah thanks, didn't notice that!